dspy/experimental/synthetic_data.py

```diff
@@ -6,21 +7,33 @@
 import dspy
 
 
-class descriptionSignature(dspy.Signature):
-  field_name = dspy.InputField(desc="name of a field")
-  example = dspy.InputField(desc="an example value for the field")
-  description = dspy.OutputField(desc="a short text only description of what the field contains")
+class DescriptionSignature(dspy.Signature):
+    """Custom signature to generate descriptions."""
```

Comment:
Docstrings are instructions when using Signatures. Notice here it’s not really an instruction.